storm-client/test/jvm/org/apache/storm/TestConfigValidate.java

```diff
@@ -541,6 +547,127 @@ public void testMapEntryCustomAnnotation() throws InvocationTargetException, NoS
         }
     }
 
+    @Test
+    public void testExactlyOneOfCustomAnnotation() throws InvocationTargetException, NoSuchMethodException, NoSuchFieldException,
```

Comment:
All the tests have it. For consistency, they should all be removed or leave them as is?

Reply:
They should be removed since they are not needed.  But we don't have to remove them from other tests in this PR since they are old code and not harmful. They can be done in a separate PR.

Reply:
Removed